conda-recipes/dotnetcore-sdk/bld.bat

```diff
@@ -0,0 +1,64 @@
+set DOTNET_ROOT=%PREFIX%\opt\dotnet
```

Comment:
Why do you need to explicitly install .netcore during the conda build? Can't you assume is already globally installed?

Reply:
This keeps with conda packaging principles that the recipe should be independent of the local environment used to build a package. As discussed offline a little while ago, this approach allows us to control precisely what version of the .NET Core SDK is used as a build dependency, with zero runtime impact.

Reply:
But we have full control of the build machine. Adding (and maintaining) this, seems a lot more work than is needed.